### ISSUE TITLE
iOS - About Comply Chain App - VoiceOver Starts Reading in Middle of Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#102]
After tapping the "About" button in the footer and then choosing the "About Comply Chain App" screen VoiceOver starts reading in the middle of the page/first paragraph. I would expect VoiceOver to focus on the top of the screen which would be the ILAB logo and not partway down the screen which means having to work their way back through the information above it to go other places. The same type of thing happens for the other screens for the other two screen too.

**Comment by jfc3-dol** [comment]
VoiceOver is still reading the screen starting in the middle, as explained above instead of at the top of the screen.

**Comment by KyleLivingstonDOL** [comment]
ZTI, can you please address alongside #101 ?

**Comment by harpreetsdev** [comment]
@jfc3-dol This one needs testing too.

@pbhatt17 @Abychalil

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

When I tested the "About Comply Chain App" from the home screen with VoiceOver, it highlights the middle of the screen, as shown in the attached screenshot.

![comply chain ios - about the comply chain app - VoiceOver starts reading where the black border is highlighting the text](https://user-images.githubusercontent.com/10423487/102369638-2b559d80-3f8a-11eb-823b-50453baa7de0.png)

I'm not sure if it's fixed for the latest version of iOS only is why it is working for you or not for me. 

Our device is on iOS 13.3.1. 

I do not want to upgrade it because if it has issues and gets bricked/does not work anymore; I don't have a device to test with then. 

In the past, I have had my personal iPhone and iPad not update properly, and I don't want to have to go to an Apple store to get it fixed during a pandemic. Plus, I'm not sure the upgrade will be able to be done on our older devices and the password to upgrade them is my boss's.

**Comment by harpreetsdev** [comment]
@jfc3-dol Thanks. Thats all right, I can understand. Although I don't know if it is just the OS update or even device dependent as me and Aby have been getting almost consistent results on different devices. Working on 12 Pro max and NOT working on iPhone 7-9. For me personally I've been getting consistent results on 12 Pro Max and 11 Pro.

@pbhatt17 @Abychalil

**Comment by jfc3-dol** [comment]
When going to a Topic screen VoiceOver reads in the middle of the page instead of the top of the screen as expected.

Please see screensot from December 16, 2020.

**Comment by nmattapalli** [comment]
On iPhone 11 Pro, ios version 14.4.1: I clicked "About" and then "About Comply Chain App" and the focus was on ILAB as in the attached picture
![102_About_focus](https://user-images.githubusercontent.com/77950769/134531661-f98c2dd3-9543-461b-8057-3bf18723c7aa.png)
.

**Comment by jfc3-dol** [comment]
@nmattapalli and @pbhatt17 

Now when I'm using VoiceOver is starts reading from the first heading at the top of the page's main content section as expected. So I'm going to close this issue.
